cspell.json

```diff
@@ -9,10 +9,17 @@
     "language": "en",
     "useGitignore": true,
     "words": [
+        "bluelightbulb",
+        "Caartaa",
         "codebases",
+        "coderepair",
+        "dtos",
+        "gppd",
```

Comment:
where did you get that from?

Reply:
https://github.com/intuita-inc/intuita-vscode-extension/blob/60fdf19d67020137688562ef10293d39f9a49a7a/test/classifier/diagnostics.txt#L2

Reply:
I see, didn't know we were checking txt files ;) all good